Server/mods/deathmatch/logic/CStaticFunctionDefinitions.cpp

```diff
@@ -1427,6 +1447,46 @@ bool CStaticFunctionDefinitions::SetElementVelocity ( CElement* pElement, const
 }
 
 
+bool CStaticFunctionDefinitions::SetElementTurnVelocity ( CElement* pElement, const CVector& vecTurnVelocity )
+{
+    assert ( pElement );
+    RUN_CHILDREN( SetElementTurnVelocity ( *iter, vecTurnVelocity ) )
+
+    int iType = pElement->GetType ();
+    switch ( iType )
+    {
+        case CElement::PED:
+        case CElement::PLAYER:
+        {
+            // TODO

```

Comment:
While setting turn speed only on the server side for peds, without saving/getting